v4-client-py/examples/composite_example.py

```diff
@@ -35,6 +35,8 @@ async def main() -> None:
         time_in_force = OrderTimeInForce[time_in_force_string]
         price = orderParams.get("price", 1350)
         time_in_force_seconds = 60 if time_in_force == OrderTimeInForce.GTT else 0
+        latest_block = client.validator_client.get.latest_block()
```

Comment:
Suggestion: the current block is technically an invalid block to use as a GTB so imo use +1
```suggestion
        latest_block = client.validator_client.get.latest_block() + 1
```